wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/Commands.java

```diff
@@ -173,8 +173,12 @@ public static Command defer(Supplier<Command> supplier, Set<Subsystem> requireme
    *
    * @param supplier the command supplier
    * @return the command
+   * @deprecated The ProxyCommand supplier constructor has been deprecated in favor of directly
```

Comment:
Can you mention that a good replacement for `deferredProxy(supplier)` is `proxy(defer(supplier)`?